libraries/chain/protocol_feature_manager.cpp

```diff
@@ -194,6 +194,18 @@ may use a new `set_proposed_producers_ex` intrinsic to access extended features.
 Builtin protocol feature: ACTION_RETURN_VALUE
 
 Enables new `set_action_return_value` intrinsic which sets a value that is included in action_receipt.
+*/
+            {}
+         } )
+
+         (  builtin_protocol_feature_t::stop_deferred_transactions, builtin_protocol_feature_spec{
+            "DEPRECATE_DEFERRED_TRANSACTIONS",
+            fc::variant("8b3d91df64471bdcf4b0f14edaf4a1cf7e082e36902a1e26516c81f6c7f658d7").as<digest_type>(),
```

Comment:
The text to be hashed should start at "Builtin protocol feature:".  You only included the description.

Reply:
Good catch. Hash has been recalculated.